simulariumio/readers/cytosim_trajectory_reader.py

```diff
@@ -0,0 +1,377 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import logging
+from typing import Dict, Any, List, Tuple
+import sys
+
+import numpy as np
+
+from .trajectory_reader import TrajectoryReader
+from ..exceptions import DataError
+
+###############################################################################
+
+log = logging.getLogger(__name__)
+
+###############################################################################
+
+
+class CytosimTrajectoryReader(TrajectoryReader):
+    def _ignore_line(self, line: str) -> bool:
+        """
+        if the line doesn't have any data, it can be ignored
+        """
+        return len(line) < 1 or line[0:7] == "warning" or "report" in line
```

Comment:
Are these all the options for non-data lines in the Cytosim files? (e.g. there is no info/error to accompany the warning?)
Also, I'm guessing there isn't a way to directly identify data lines as opposed to disqualifying other line types?

Reply:
yeah this is based on what I've seen in the test files I've been working with, there's no documented standard that I've found